code/game/objects/items/binoculars.dm

```diff
@@ -27,13 +27,14 @@
 	var/cooldown_duration = 200 //20 seconds
 	var/obj/effect/overlay/temp/laser_target/laser
 	var/target_acquisition_delay = 100 //10 seconds
-	var/mode = 0 //Able to be switched between modes, 0 for cas laser, 1 for finding coordinates, 2 for directing railgun, 3 for orbital bombardment.
+	var/mode = 0
 	var/changable = TRUE //If set to FALSE, you can't toggle the mode between CAS and coordinate finding
 	var/ob_fired = FALSE // If the user has fired the OB
 	var/turf/current_turf // The target turf, used for OBs
 	///Last stored turf targetted by rangefinders
 	var/turf/targetturf
 
+	var/linked_mortar
```

Comment:
You either use a reference and you directly call proc, or you do everything with signals.

Both works, I think doing everything with signal is better but it's no big deal.

If you want to keep the linked_mortar ref, you have to make sure that it is correctly cleaned (aka, linked_mortar = null), when the mortar is destroyed

And don't forget autodoc